Transaction.cs

```diff
@@ -0,0 +1,37 @@
+﻿public class Transaction
+{
+    private string date;
+    private string from;
+    private string to;
+    private string narrative;
+    private decimal amount;
+
+    public Transaction(string date, string from, string to, string narrative, decimal amount)
+    {
+        this.date = date;
+        this.from = from;
+        this.to = to;
+        this.narrative = narrative;
+        this.amount = amount;
+    }
+
+    public string GetSummary()
+    {
+        return string.Join(" ", new string[] {date, narrative});
```

Comment:
If you're just joining two strings, then there's no need to create a list so that you can use `string.Join`. Instead you can just use string interpolation:
```
$"{date} {narrative}"
```
🙂